datumaro/util/meta_file_util.py

```diff
@@ -38,29 +36,3 @@ def parse_meta_file(path):
             label_map[label] = tuple(colors[int(i)])
 
     return label_map
-
-def save_meta_file(path, categories):
-    dataset_meta = {}
-
-    label_map = {}
-    for i, label in enumerate(categories[AnnotationType.label]):
-        label_map[str(i)] = label.name
-    dataset_meta['label_map'] = label_map
-
-    if categories.get(AnnotationType.mask):
-        bg_label = find(categories[AnnotationType.mask].colormap.items(),
-            lambda x: x[1] == (0, 0, 0))
-        if bg_label is not None:
-            dataset_meta['background_label'] = str(bg_label[0])
-
-        segmentation_colors = []
-        for color in categories[AnnotationType.mask].colormap.values():
-            segmentation_colors.append([int(color[0]), int(color[1]), int(color[2])])
-        dataset_meta['segmentation_colors'] = segmentation_colors
-
-    meta_file = path
-    if osp.isdir(path):
-        meta_file = get_meta_file(path)
-
-    with open(meta_file, 'w') as f:
-        json.dump(dataset_meta, f)
```

Comment:
I think the implementation itself can be left here, not to be spread over multiple files.

Reply:
Fixed.